dreamhome.py

```diff
@@ -1,16 +1,46 @@
-from flask import Flask
+from flask import Flask, jsonify
 from database.db import initialize_db
+from api.routes.api_routes import api
+from api.helpers.errors import APIError
+import traceback
+import pry
 
-app = Flask(__name__)
+app = Flask(__name__, template_folder = 'api/views')
 
-app.config['MONGODB_SETTINGS'] = {
-    'host': 'mongodb://localhost/dreamhome'
-}
+app.config.from_envvar('ENV_FILE_LOCATION')
 
+@app.route('/')
+def hello():
+    return 'Welcome to your dream home server!'
+
+app.register_blueprint(api)
 initialize_db(app)
 
-@app.route("/")
-def hello():
-    return {"hello": "world"}
```

Comment:
Bye Bye @app.route("/")

@priyapower, this is what you said was moved to a routes file outside of api_routes?

Reply:
I believe that we should have this route in a separate routes file from our apis and associated with a Welcome or Documentation Controller or something. I don't know if this has happened because I still see it on ln12 of `dreamhome.py`

Reply:
I don't believe that happened.  We should spend some time grooming more documentation.